charts/values.yaml

```diff
@@ -216,6 +220,11 @@ apiServer:
   tolerations: [ ]
     # - key: node-role.kubernetes.io/master
     #   operator: Exists
+  ## @param apiServer.serviceType default service type for apiserver
+  ## "LoadBalancer" means using LoadBalancer
+  ## "ClusterIP" means using ClusterIP
+  ## "NodePort" means using NodePort
+  serviceType: ClusterIP
```

Comment:
Is it possible to custom the port of the nodeport?

Reply:
I think that `NodePort` just for testing in development, so I let it be allocated automatically. If you think it is necessary, I can add this.

Reply:
Configurable will support more scenes, automatically assign if it is not set. :)

Reply:
I have added it.